api/IDBVersionChangeEvent.json

```diff
@@ -9,7 +9,7 @@
               "version_added": "24"
             },
             {
-              "version_added": "23",
+              "version_added": "12",
```

Comment:
You have a good point there, maybe this should be something the consistency check tests against.  I noticed that in api/RTCPeerConnection.json with Safari.  Perhaps we should incorporate that check into the consistency tests (though maybe in the future, since these changes have already gotten so huge).

Reply:
As I haven't fully reviewed your latest consistency check code, it would be nice if you could write down somewhere what you already test and what we want to test later, so that we can track all these linting ideas better.

Reply:
Of course!  I'll write it down in the main PR.  :wink: